syndesis/plugins/org.fusesource.ide.syndesis.extension.core/src/org/fusesource/ide/syndesis/extensions/core/model/SyndesisAction.java

```diff
@@ -15,14 +15,60 @@
 import java.util.List;
 import java.util.Map;
 
+import org.fusesource.ide.foundation.core.databinding.PojoModelObservable;
+
 import com.fasterxml.jackson.annotation.JsonAnyGetter;
 import com.fasterxml.jackson.annotation.JsonAnySetter;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 
 /**
  * @author lheinema
  */
-public class SyndesisAction {
+@JsonPropertyOrder({ "id", "name", "description", "descriptor", "tags", "actionType" })
```

Comment:
this order is a constraint from the schema or it is one from our model?

Reply:
it comes from the model classes used in syndesis runtime